go/obscuronode/enclave/enclave.go

```diff
@@ -391,6 +380,48 @@ func (e *enclaveImpl) IsInitialised() bool {
 	return e.storage.FetchSecret() != nil
 }
 
+func (e *enclaveImpl) isGenesisBlock(block *types.Block) bool {
+	return e.l1Blockchain != nil && block.Hash() != e.l1Blockchain.Genesis().Hash()
```

Comment:
Just a curiosity, what would happen after a restart? The `l1Blochkcian` will be recreated in the constructor I guess.

Reply:
Yeah, that's right.